cloudinit/config/cc_fan.py

```diff
@@ -35,6 +35,7 @@
     "description": MODULE_DESCRIPTION,
     "distros": distros,
     "frequency": PER_INSTANCE,
+    "activate_by_schema_keys": [],
```

Comment:
I think we need "fan" here because BUILTIN_CFG has `"config": None` which would [default result in skipping](https://github.com/canonical/cloud-init/blob/main/cloudinit/config/cc_fan.py#L93)

Reply:
done